flink-connector-vitess-cdc/src/main/java/com/ververica/cdc/connectors/vitess/table/VitessTableSource.java

```diff
@@ -212,8 +240,18 @@ public String toString() {
                 + ", tableName='"
                 + tableName
                 + '\''
-                + ", vtctldConfig="
-                + vtctldConfig
+                + ", shard='"
+                + shard
```

Comment:
These nullable fields will cause the NullPointerException. Please use like this.
```java
String.valueOf(shard)
```